app/controllers/api/v1/customers/favorite_merchant_controller.rb

```diff
@@ -1,5 +1,4 @@
 class Api::V1::Customers::FavoriteMerchantController < ApplicationController
-  respond_to :json
```

Comment:
I took this out of every controller and put it inside of `app/controllers/application_controller.rb` so we don't have to have it at the top of every controller